recipe/meta.yaml

```diff
@@ -1,44 +1,64 @@
-{% set version = "2018_20171205" %}
+{% set vmajor = 2018 %}       # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MAJOR +(\d+)").group(1)|int
+{% set vminor = 0 %}          # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_VERSION_MINOR +(\d+)").group(1)|int
+{% set vinterface =  10003 %} # load_file_regex(load_file='include/tbb/tbb_stddef.h', regex_pattern="TBB_INTERFACE_VERSION +(\d+)").group(1)|int
+{% set version = "%d.%d.%d"%(vmajor,vminor,vinterface%1000) %}
+{% set args=' --build-args=stdver=c++11 --install-libs --install-devel --install-docs' %} #  no --install-python yet
 
 package:
   name: tbb
   version: {{ version }}
 
 source:
-  fn: tbb{{ version }}oss_src.tgz
-  url: https://github.com/01org/tbb/archive/2018_U2.tar.gz
-  sha256: 78bb9bae474736d213342f01fe1a6d00c6939d5c75b367e2e43e7bf29a6d8eca
+  git_url: https://github.com/01org/tbb.git
+  git_tag: {{ vmajor }}_U{{ vinterface%1000 }}
 
 build:
-  number: 0
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  script: python build/build.py {{ args }}                                          # [unix]
+  script: python build/build.py --make-tool=mingw32-make {{ args }}                 # [win]
+  #script: env MACOSX_DEPLOYMENT_TARGET=10.7 python build/build.py {{ args }}       # [osx]
+  script_env:
+    - OS    # [win]
   features:
-    - vc9  # [win and py27]
+    - vc9   # [win and py27]
     - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  entry_points:
+  #  - python-tbb = tbb:_main # for tb4py
```

Comment:
Same story likely with the empty `entry_points`. Drop?

Reply:
Trying to following the suit as much as possible, https://github.com/conda-forge/tbb-feedstock/pull/24
still no luck